### PR TITLE
[Python] Add support for EXPLAIN ANALYZE to `explain` method

### PR DESCRIPTION
**Pull request by Tishj**
By accepting a "type" argument in `explain`, we can now support analyze (type defaults to "standard")
```py
>>> print(duckdb.sql('select 42').explain("analyze"))
┌─────────────────────────────────────┐
│┌───────────────────────────────────┐│
││    Query Profiling Information    ││
│└───────────────────────────────────┘│
└─────────────────────────────────────┘

┌─────────────────────────────────────┐
│┌───────────────────────────────────┐│
││        Total Time: 0.0030s        ││
│└───────────────────────────────────┘│
└─────────────────────────────────────┘
┌───────────────────────────┐
│      RESULT_COLLECTOR     │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│             0             │
│          (0.00s)          │
└─────────────┬─────────────┘                             
┌─────────────┴─────────────┐
│      EXPLAIN_ANALYZE      │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│             0             │
│          (0.00s)          │
└─────────────┬─────────────┘                             
┌─────────────┴─────────────┐
│         PROJECTION        │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│             42            │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│             1             │
│          (0.00s)          │
└─────────────┬─────────────┘                             
┌─────────────┴─────────────┐
│         DUMMY_SCAN        │
│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
│             1             │
│          (0.00s)          │
└───────────────────────────┘  
```

### CODE REVIEW

**Comment by Tishj** [comment]
Maybe it's preferable to turn this into an enum instead, so we can add stubs - and IDEs will offer auto suggestion/completion for it

**Comment by Tishj** [comment]
it now accepts either:
```py
print(duckdb.sql('select 42').explain(duckdb.EXPLAIN_ANALYZE))
```
or
```py
print(duckdb.sql('select 42').explain('analyze'))
```
or
```
print(duckdb.sql('select 42').explain(1))
```

This approach of accepting an `int` and creating constants for them is taking from `csv`
I also considered making a separate class for it that contains these constants, but I think that even feels even clunkier

File: tools/pythonpkg/duckdb_python.cpp [hunk 1]
```diff
@@ -207,6 +207,9 @@ PYBIND11_MODULE(DUCKDB_PYTHON_LIB_NAME, m) {
 	m.attr("threadsafety") = 1;
 	m.attr("paramstyle") = "qmark";
 
+	m.attr("EXPLAIN_STANDARD") = 0;
```
Comment (Mause) [inline tools/pythonpkg/duckdb_python.cpp:210]: Why not use `py::enum_` here?

**Comment by Mause** [comment]
> Maybe it's preferable to turn this into an enum instead, so we can add stubs - and IDEs will offer auto suggestion/completion for it

You can technically use strings as an enum like thing, and give good types, using `typing.Literal["all", "minimal", "none", "nonnumeric"]` or similar

File: tools/pythonpkg/duckdb_python.cpp (hunk 1 above)
Comment (Tishj) [reply tools/pythonpkg/duckdb_python.cpp:210]: That's also possible, but like I said, that introduces another level:
```py
print(duckdb.sql('select 42').explain(duckdb.ExplainType.ANALYZE))
```

Comment (Tishj) [reply tools/pythonpkg/duckdb_python.cpp:210]: Oh I actually that's not true
```py
>>> import duckdb 
>>> duckdb.ANALYZE
<ExplainType.ANALYZE: 1>
```

**Comment by Tishj** [comment]
Test failure is unrelated, relates to #6582

**Comment by Mytherin** [comment]
Thanks! LGTM

**Pull request merged**